scalameta/contrib/shared/src/main/scala/scala/meta/contrib/DocToken.scala

```diff
@@ -264,11 +264,6 @@ object DocToken {
   /**
     * Documents a Scaladoc Heading.
     */
-  case object Heading extends Kind
```

Comment:
Since the number of headings is limited, can we use multiple case objects?

```scala
abstract class Heading(val level: Int) extends Kind
case object Heading1 extends Heading(1)
case object Heading2 extends Heading(2)
...
```
Sure, it a bit of boilerplate but it avoids validating false input like `Heading(-1)`.